### PR TITLE
chore(Upload): add events

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/components/upload/types.ts [hunk 1]
```diff
@@ -30,6 +30,16 @@ export type UploadProps = {
    */
   fileMaxSize?: number
 
+  /**
+   * will be called on `files` changes made by the user. Access the files with `{ files }`.
+   */
+  onChange?: ({ files }: { files: UploadFile[] }) => void
+
+  /**
+   * will be called once a file gets deleted by the user. Access the deleted file with `{ file }`.
```
Comment (langz) [inline packages/dnb-eufemia/src/components/upload/types.ts:39]: I see that in our docs, `upload/events.md`, we write _Access the deleted file with `{ fileItem }`._

**Review by langz — APPROVED** [review]
👏

File: packages/dnb-eufemia/src/components/upload/types.ts (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/upload/types.ts:39]: `file` is not god, because it contains `file` inside as well.

Comment (tujoworker) [reply packages/dnb-eufemia/src/components/upload/types.ts:39]: Ohh, yes, will fix it 👍

**Pull request merged**